src/query/functions/src/scalars/array.rs

```diff
@@ -530,4 +535,70 @@ pub fn register(registry: &mut FunctionRegistry) {
             lhs.iter().contains(&rhs)
         }),
     );
+
+    registry.register_passthrough_nullable_1_arg::<EmptyArrayType, UInt64Type, _, _>(
+        "array_unique",
+        FunctionProperty::default(),
+        |_| FunctionDomain::Domain(SimpleDomain { min: 0, max: 0 }),
+        vectorize_1_arg::<EmptyArrayType, UInt64Type>(|_, _| 0),
```

Comment:
duplicated registers for  `EmptyArrayType`

Reply:
we need this registers, otherwise `array_unique([])` will return error.
```
mysql> select array_unique([]);
ERROR 1105 (HY000): Code: 1065, displayText = no overload satisfies `array_unique(Array(Nothing))`

has tried possible overloads:
  array_unique(Array(T0)) :: UInt64            : unbound generic type `T0`
  array_unique(Array(T0) NULL) :: UInt64 NULL  : unbound generic type `T0`.
```